IdentityServer/v6/docs/content/quickstarts/4_ef.md

```diff
@@ -4,117 +4,193 @@ date: 2020-09-10T08:22:12+02:00
+                sql => sql.MigrationsAssembly(migrationsAssembly));
+        })
+        .AddTestUsers(TestUsers.Users);
+    
+    //...
+}
+```
 {{% notice note %}}
-You can find the latest SQL scripts for SqlServer in our EF [repository](https://github.com/DuendeSoftware/IdentityServer/tree/main/migrations/IdentityServerDb/Migrations).
+
+You will use Entity Framework migrations later on in this quickstart to manage
+the database schema. The call to *MigrationsAssembly(...)* tells Entity
+Framework that the host project will contain the migrations. This is necessary
+since the host project is in a different assembly than the one that contains the
+*DbContext* classes.
+
 {{% /notice %}}
 
-### Configuring the Stores
-To start using these stores, you’ll need to replace any existing calls to *AddInMemoryClients*, *AddInMemoryIdentityResources*, *AddInMemoryApiScopes*, *AddInMemoryApiResources*, and *AddInMemoryPersistedGrants* in your *ConfigureServices* method in *Startup.cs* with *AddConfigurationStore* and *AddOperationalStore*.
+## Managing the Database Schema
 
-These methods each require a *DbContextOptionsBuilder*, meaning your code will look something like this:
+The *Duende.IdentityServer.EntityFramework.Storage* Nuget package (installed as
+a dependency of *Duende.IdentityServer.EntityFramework*) contains entity classes
+that map onto IdentityServer's models. These entities are maintained in sync
+with IdentityServer's models - when the models are changed in a new release,
+corresponding changes are made to the entities. As you use IdentityServer and
+upgrade over time, you are responsible for your database schema and changes
+necessary to that schema.
 
-```cs
-var migrationsAssembly = typeof(Startup).GetTypeInfo().Assembly.GetName().Name;
-const string connectionString = @"Data Source=Duende.IdentityServer.Quickstart.EntityFramework.db";
+One approach for managing those changes is to use [EF
+migrations](https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/index),
+which is what this quickstart will use. If migrations are not your preference,
+then you can manage the schema changes in any way you see fit. 
 
-services.AddIdentityServer()
-    .AddTestUsers(TestUsers.Users)
-    .AddConfigurationStore(options =>
-    {
-        options.ConfigureDbContext = b => b.UseSqlite(connectionString,
-            sql => sql.MigrationsAssembly(migrationsAssembly));
-    })
-    .AddOperationalStore(options =>
-    {
-        options.ConfigureDbContext = b => b.UseSqlite(connectionString,
-            sql => sql.MigrationsAssembly(migrationsAssembly));
-    });
-```
+{{% notice note %}}
 
-You might need these namespaces added to the file:
+You can find the latest SQL scripts for SqlServer in our EF
+[repository](https://github.com/DuendeSoftware/IdentityServer/tree/main/migrations/IdentityServerDb/Migrations).
 
-```cs
-using Microsoft.EntityFrameworkCore;
-using System.Reflection;
+{{% /notice %}}
```

Comment:
Here's our "docs" for the same topic: https://docs.duendesoftware.com/identityserver/v6/data/ef/#database-creation-and-schema-changes-across-different-versions-of-identityserver

We've had people confused that these SQL scripts are "official" so we want to reinforce here or above or elsewhere that it's their SQL scripts they should use (however they create and manage them over time when schema changes occur).

Reply:
Let's just cut this note entirely. We don't need it for the quickstart, and we have it documented elsewhere. The section above is reinforcing that the db schema is the user's responsibility.